src/install/installMultiple.ts

```diff
@@ -385,6 +385,7 @@ async function install (
 
     const peerDependencies = peerDependenciesWithoutOwn(pkg)
 
+    const nodeMajor = process.version.substring(0, process.version.indexOf('.'))
```

Comment:
move this line to the beginning of the file. It is enough to calculate nodeMajor once. There is no need to recalculate it on each function call